frontend/src/pages/org/index.ts

```diff
@@ -82,6 +88,17 @@ export class Org extends LiteElement {
   @state()
   private isSavingOrgName = false;
 
+  @state()
+  private isFetching = false;
+
+  @state()
+  private fetchErrorStatusCode?: number;
+
+  // Use to cancel requests
+  private getStorageQuotaReachedController: AbortController | null = null;
+
+  private timerId?: number;
```

Comment:
To prevent lit from performing updates after the component is removed from the DOM, you can clear the timer in `disconnectedCallback()`:
```
  disconnectedCallback() {
    window.clearInterval(this.timerId);
    super.disconnectedCallback();
  }
  ```